src/main/java/org/openrewrite/github/AddCronTrigger.java

```diff
@@ -17,31 +17,63 @@
 
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import org.jetbrains.annotations.VisibleForTesting;
 import org.openrewrite.Option;
 import org.openrewrite.Recipe;
 import org.openrewrite.yaml.MergeYaml;
 
 import java.time.Duration;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
 
 @EqualsAndHashCode(callSuper = false)
 @Getter
 public class AddCronTrigger extends Recipe {
     @Option(displayName = "Cron expression",
-            description = "Using the [POSIX cron syntax](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07).",
-            example = "0 18 * * *")
+            description = "Using the [POSIX cron syntax](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07) or the non standard options" +
+                    " @daily @weekly @monthly @hourly @yearly.",
+            example = "@daily")
     private final String cron;
 
-    public AddCronTrigger(String cron) {
-        this.cron = cron;
+    @VisibleForTesting
+    final transient Random random;
+
```

Comment:
Jackson was complaining about this. Custom Random instances are only used within tests and I am assuming that `transient` is OK